### ISSUE TITLE
Icon breakage / mismatch

### ISSUE THREAD

**Issue by countermeasure** [jquense/react-widgets#145]
On upgrading from v2.4.1 to v2.7.1, `rw-widgets.woff` icons have broken for me.

Specifically:
1. `rw-i-caret-left` now looks like caret-down
2. `rw-i-caret-down` no longer renders, and
3. `rw-i-calendar` no longer renders.

This commit looks like it might be involved: a7a63a5

When I roll back to v2.4.1, everything is fine. When I go forward to v2.7.1, icons break again.

Not sure if this is somehow my fault, or a bug. Letting you know in case it's the latter.

Loving your library!

**Comment by jquense** [comment]
Hi there. 

it's a bit hard to guess without knowing about how you are loading the styles, but It's probably the case that you are loading the old icon files with the newer style sheet. be sure to clear your caches and re copy the fonts and images

**Comment by countermeasure** [comment]
I've checked that after upgrading to 2.7.1:
- old copies of the icon files have been replaced with new ones, and
- the browser's cache has been cleared.

I'm reasonably inexperienced, but I've made as certain as I can that I've done both these things correctly. The problem persists.

I'm loading the styles with `less` like so:
```
@import "../../node_modules/react-widgets/lib/less/react-widgets.less";
```

Not asking you to diagnose, just letting you know in case it's a bug and not my error. I suspect it's my error though!

**Comment by mathieumg** [comment]
Are you also loading Font-Awesome? If you are and you do it after React-Widgets, it might be cause the `fa-css-prefix` gets overridden with Font-Awesome's one. See: https://github.com/jquense/react-widgets/pull/142

**Comment by jquense** [comment]
there was also a recent set of bugs in webpack s css-loader if you are using webpack which was breaking icons. try upgrading it to the latest, or downgrading to 0.14. other than that I'm not sure, I am pretty sure it's not a rw bug, but I could always be wrong. :)

**Comment by mathieumg** [comment]
FWIW, the latest tag in https://github.com/webpack/css-loader/releases (0.15.1) doesn't cover the fix to the icon problems, but since he said it worked again when downgrading back to an older React-Widgets release, I don't think the problem stems from that.

**Comment by jquense** [comment]
I think the font awesome thing is a good guess as well, tho I am not sure I can fix it right away since it strikes meas a breaking change. Perhaps we need to put a workaround somewhere for that if that is the issue

**Comment by countermeasure** [comment]
I am using Font Awesome, although I'm loading it before React Widgets.

I'm not using WebPack.

Given that in 2.7.1 I see a down-caret (was e801, now e803) where I should see a left-caret (was e807, now e801), there's no chance that `rw-widgets.woff` wasn't successfully updated during that change?

That's the only idea I have. Sorry not to be able to be more helpful.

**Comment by jquense** [comment]
files on my end are all updated, and I know multiple people using `2.7.1` without this issue...it really sounds like your font files are out of date, either cached to aggressively somewhere or they din't get downloaded correctly...

how are you getting the files? npm, bower? can you try completely deleting the files and re installing everything/

**Comment by countermeasure** [comment]
It does sounds like it's a problem that I've made then.

I'm installing via npm.

I actually uninstalled all my Node modules, cleared the Node cache, then reinstalled everything to see if that fixed it, but no dice. I'm probably overlooking something basic, and no doubt I'll get it figured out over time.

I raised this issue because I thought on the off-chance that this was a bug I should let you know. Sounds like it's not a bug though, so feel free to close it.

Thanks for taking the time to help me try to sort it out!
